tests/service/test_autotune_service.py

```diff
@@ -51,7 +52,7 @@ def __init__(
         self.tensor_list = tensor_list
         self.client = AutotuneClient(service_addr, service_port)
 
-    def run(self, total_iters=5000):
+    def run(self, total_iters=500000):
```

Comment:
is the test deterministic now or you are trying to get a better result with more iterations?

Reply:
test is deterministic, but need more iters to make sure autotune finishes.